__tests__/endpoints.test.js

```diff
@@ -62,6 +62,58 @@ describe('ERRORS - /api/topics', () => {
   });
 });
 
+describe('GET /api/articles', () => {
+  it('should return an array of all articles', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.articles)).toBe(true);
+        expect(body.articles.length).toBeGreaterThan(0);
+        expect(body.articles[0]).toHaveProperty('author');
+        expect(body.articles[0]).toHaveProperty('title');
+        expect(body.articles[0]).toHaveProperty('article_id');
+        expect(body.articles[0]).toHaveProperty('topic')
+        expect(body.articles[0]).toHaveProperty('created_at')
+        expect(body.articles[0]).toHaveProperty('votes');
+        expect(body.articles[0]).toHaveProperty('article_img_url')
+        expect(body.articles[0]).toHaveProperty('comment_count')
+      });
+  });
+  it('200 - responds with articles sorted by date in descending order', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        const sortedArticles = body.articles.map(article => ({
+          ...article,
+          created_at: new Date(article.created_at)
+        }));
+        expect(sortedArticles).toBeSortedBy('created_at', { descending: true });
+      });
+  });
```

Comment:
You can simplify this test by removing the spread operation here and just check the order 😄